### PR TITLE
Hide readviz by default

### LINKED ISSUE #633: Hide readviz by default?
BAI files for v3.1 are very large (~6 MB) and more than one may be loaded for a variant. Consider hiding the reads data section by default to avoid downloading so much data when it may not be needed.

Ideally, there should be an option to always show readviz to support users that frequently look at reads.

### PR DESCRIPTION
**Pull request by rileyhgrant**
Resolves #633 

Hides readviz by default on Variant pages to try and save on egress fees.

Gives two options,
- a button to load the data just this time
- a checkbox to always load readviz data, this uses `localStorage` to save the value of this on load and when toggling this

Also adds tracking to these two buttons to try and see how often people use these

### CODE REVIEW

File: browser/src/ReadData/ReadData.tsx [hunk 1]
```diff
@@ -307,8 +325,8 @@ class ReadData extends Component<ReadDataProps, ReadDataState> {
   }
 
   loadInitialTracks() {
-    ;['exome', 'genome'].forEach((exomeOrGenome) => {
-      ;['het', 'hom', 'hemi'].forEach((category) => {
+    ;(['exome', 'genome'] as SequencingType[]).forEach((exomeOrGenome) => {
```
Comment (phildarnowsky-broad) [inline browser/src/ReadData/ReadData.tsx:343]: if this works, it'd be more idiomatic to write `['exome', 'genome'].forEach((exomeOrGenome: SequencingType) => ...`. I _think_ TS should be able to infer that properly. Same deal in the next line.

File: browser/src/ReadData/ReadData.tsx [hunk 2]
```diff
@@ -517,102 +541,126 @@ const ReadDataContainer = ({ datasetId, variantIds }: ReadDataContainerProps) =>
...
+        <Button id="load-once" disabled={loadOnce} onClick={() => setLoadOnce(!loadOnce)}>
+          Load once
+        </Button>
+        <div>
+          <br />
+          <span>
+            <CheckboxInput
+              id="always-load"
+              checked={alwaysLoad}
+              onChange={(e: any) => {
```
Comment (phildarnowsky-broad) [inline browser/src/ReadData/ReadData.tsx]: there should be a type you can use for `e` here, `MouseEvent` or something similar IIRC

**Review by phildarnowsky-broad — CHANGES_REQUESTED** [review]

File: browser/src/ReadData/ReadData.tsx (hunk 1 above)
Comment (rileyhgrant) [reply browser/src/ReadData/ReadData.tsx:343]: I would have liked that better too, but unfortunately TS complains about doing it that way, as the `forEach` is being called on an array of strings.

**Comment by rileyhgrant** [comment]
Screenshots of the UI: 

- Screenshot 2024-06-17 at 15 48 28
- Screenshot 2024-06-17 at 15 48 17
- Screenshot 2024-06-17 at 15 48 49

**Review by phildarnowsky-broad — APPROVED** [review]
Cut, print, that's a wrap

**Review by sjahl — APPROVED** [review]
This looks good to me from a UI / Text copy perspective

**Pull request merged**